.github/workflows/validate-nginx-config.yml

```
@@ -0,0 +1,78 @@
+      PORT: ${{ inputs.port }}
+      NGINX_ENVSUBST_TEMPLATE_DIR: ${{ inputs.templatedir}}
+    defaults:
+      run:
+        working-directory: ${{ inputs.workdir }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: check Nginx version
+        run: sudo nginx -v
+        if: ${{ inputs.verbose }}
+      - name: Parse custom headers and export as environment variables
+        run: |
```

actionlint (1.7.4)

property "custom_headers" is not defined in object type {custom_template_values: string; port: number; templatedir: string; verbose: bool; workdir: string}

(expression)